sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -784,21 +784,17 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
   }
 
   def renameTempViewToExistingViewError(oldName: String, newName: String): Throwable = {
-    new AnalysisException(
-      errorClass = "_LEGACY_ERROR_TEMP_1062",
```

Comment:
If you remove this error classes, please, remove it from `error-classes.json` too.

Reply:
Done